ansible-playbooks/roles/deploy/vars/main.yml

```diff
@@ -0,0 +1,9 @@
+ENV: ""
```

Comment:
Ok, this here makes my earlier comment not valid, because this defines each value, and sets them to "". These variables will be defined but empty.